src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
 // write your code here
+const promise1 = new Promise((resolve, reject) => {
+  // додаю обробник кліку на елемент з класом лого
+  const logoElement = document.querySelector('.logo');
+
+  if (logoElement) {
+    logoElement.addEventListener('click', () => {
+      resolve('First promise was resolved');
+    });
+  } else {
+    reject(new Error('First promise not was resolved'));
+  }
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+promise1
+  .then((message) => {
+    const div = document.createElement('div');
+
+    div.classList.add('message');
+    div.textContent = message;
+
+    document.body.appendChild(div);
+  })
+  .catch((error) => {
+    const div = document.createElement('div');
+
+    div.classList.add('message', 'error-message');
+    div.textContent = error;
```

Comment:
When setting the text content of the error message, it might be more informative to use `error.message` instead of `error` to ensure the error message is displayed correctly.